health/health.d/geth.conf

```diff
@@ -0,0 +1,23 @@
+ template: chainhead_diff_between_header_block
+       on: geth.chainhead
+    class: Workload
+     type: Go.d plugin;
```

Comment:
@ilyam8 , please, take a look here. I do not remember to see go.d.plugin written this way. :thinking:

Reply:
Just let me merge😔

Reply:
@OdysLam please, check [alarm-line-type](https://learn.netdata.cloud/docs/agent/health/reference#alarm-line-type) description, it is 

> Type can be used to indicate the broader area of the system that the alarm applies to. For example, under the general Database type, you can group together alarms that operate on various database systems, like MySQL, CockroachDB, CouchDB etc.

@thiagoftsm is correct.

Reply:
Also, whatever the `type` is, don't add `;`